guides/common/modules/con_migrating-project-to-el8.adoc

```diff
@@ -0,0 +1,35 @@
+[id="migrating-project-to-el8_{context}"]
+ifdef::satellite[]
+= Migrating {ProjectName} to {RHEL} 8
+endif::[]
+ifndef::satellite[]
+= Migrating {ProjectName} to Enterprise Linux 8
+endif::[]
+
+When you migrate your {Project}, you create a backup of your {ProjectServer} and your {SmartProxy}, install a fresh instance on a new server, and restore your backup on the new server.
+After your migration is complete, you can then decommission the earlier version of {ProjectServer} and {SmartProxy}.
+
+.Terminology
+Ensure that you understand the following terms:
+
+*Source server*:: The origin of migration on which you create a backup.
+
+*Target server*:: The new server on which you restore the backup.
+
+.High-Level Procedure
+To migrate your {Project} to new hardware, follow these high-level steps:
+
+. Create a backup of the {ProjectServer} or {SmartProxyServer} on the source server.
+. Perform a fresh installation of the {ProjectServer} or {SmartProxyServer} on a target server.
+ifdef::satellite[]
+* Install a minimal {RHEL} 8 instance with the capacity to store backup files.
+endif::[]
+ifndef::satellite[]
+* Install a minimal Enterprise Linux 8 (CentOS Stream 8, {RHEL} or a {RHEL} rebuild) instance with the capacity to store backup files.
```

Comment:
```suggestion
* Install a minimal Enterprise Linux 8 (CentOS Stream, {RHEL} or a {RHEL} rebuild) instance with the capacity to store backup files.
```

It's weird to say "EL 8 (CentOS Stream 8, RHEL without version, rebuild without version)". I think it's sufficient to have the 8 outside the parenthesis (or you need to say "CentOS Stream 8, RHEL8, RHEL8 rebuild").